src/analyz/mod.rs

```diff
@@ -3,16 +3,15 @@
 use rustc_ast::{ast, token, tokenstream, visit, ptr, Crate};
```

Comment:
I'm going to start working on the name resolution errors in `mod.rs`, specifically the two small batches of `mod.rs` errors currently at the top of `cargo check --message-format short`

Reply:
Done (this turned out to be many instances of the same few issues)